tspng/extraction.py

```diff
@@ -13,3 +13,16 @@ def extract_from_file(path, mime_type=MIME_TYPE):
     #load
     dict=json.loads(meta[mime_type])
     return dict
+
+def extract_from_files(paths, mime_type=MIME_TYPE):
+    nested_dict = {}
+    for path in paths:
+        #open
```

Comment:
Can this be refactored to use the `extract_from_file` function? Like so,

```python
def extract_from_files(paths, mime_type=MIME_TYPE):
    nested_dict = {}
    for path in paths:
        nested_dict[path] = extract_from_file(path, mime_type)
    return nested_dict
```

Reply:
Change made as requested, thanks!